mediaqueries-5/Overview.bs

```diff
@@ -2673,10 +2675,15 @@ Detecting the desire for increased or decreased color contrast from elements on
 				or some other arrangement that is neither particularly low or high contrast.
 			</div>
 
-			Note: When the User Agent can determine whether the particular forced color scheme
-			chosen by the user is high or low contrast,
-			one of ''prefers-contrast: more'' or ''prefers-contrast: less'' should match,
+			Additionally, when the user agent can determine
+			whether the forced color palette chosen by the user
+			has a high or low contrast,
+			one of ''prefers-contrast: more'' or ''prefers-contrast: less'' should match
```

Comment:
```suggestion
			one of ''prefers-contrast: more'' or ''prefers-contrast: less'' must match
```

Reply:
Given that we're not setting an explicit criteria by which to judge whether the forced color palette chosen by the user has a high or low contrast, I don't know if there's really a usable difference between must and should. I went with should be cause if the browser doesn't do it, it's not observable whether that is because it did not judge the contrast to be high/low, or because it did, but chose not to make the prefers-contrast query match.

I figured that if we cannot tell the two apart, insisting on a must didn't buy us anything.

I don't necessarily oppose switching to must, but can you help me see how that helps?